win32.js

```diff
@@ -44,8 +44,14 @@ module.exports = {
       if (opts.appCopyright) {
         rcOpts['version-string'].LegalCopyright = opts.appCopyright
       }
+      
+      // add "asInvoker" to the built manifest by default
+      rcOpts['requested-executation-level'] = 'asInvoker'
+      if (opts.win32exeLevel) {
```

Comment:
Electron apps on Windows have a .NET assembly?

Those other settings sound like things that Electron should already set, but supportedOS might be useful if the app depends on another exe with a higher Windows requirement...